docs/02-architecture/06-client-side-architecture.md

```diff
@@ -316,6 +312,10 @@ There are five projects in the Javascript architecture:
 - [Facia](https://github.com/guardian/frontend/tree/master/static/src/javascripts/projects/facia) - Contains JS modules and views for the weather, snaps and fronts containers
 - [Membership](https://github.com/guardian/frontend/tree/master/static/src/javascripts/projects/membership) - Contains the formatters, payment and stripe javascripts
 
+### Components
+
+[Here be 3rd-party JS](https://github.com/guardian/frontend/tree/master/static/vendor/javascripts/components) such as bean, bonzo and fastdom.
+
 ### Vendor
 
-Contains [vendor JS](https://github.com/guardian/frontend/tree/master/static/src/javascripts/vendor) from the likes of forsee, formstack and stripe.
+Contains [vendor JS](https://github.com/guardian/frontend/tree/master/static/vendor/javascripts) from the likes of forsee, formstack and stripe.
```

Comment:
we currently have `vendor` which is files we've just committed, and `components`, which is files we've pulled out of `bower_components` and committed. i don't really understand why, it's historical and seems weird, but i'd rather fix that in another PR another time (by dropping bower)